iscsi/iscsi.go

```diff
@@ -203,7 +205,8 @@ func getMultipathDisk(path string) (string, error) {
 }
 
 // Connect attempts to connect a volume to this node using the provided Connector info
-func Connect(c Connector) (string, error) {
+func Connect(c *Connector) (string, error) {
```

Comment:
Given that we're not modifying Connector why would you convert it to a pointer here?  It's a relatively light struct, is there something you're thinking that would be handy to use it for persistence going forward maybe?

Reply:
I don't have any reason it should be mutable.  I think I was just in the habit of passing pointers rather than values to avoid copies.  Since Connector calls aren't a tight loop (or at least shouldn't be) I'm fine with it being a copy-call.  I'll post and update changing it.